src/pkg/user/service/service.go

```diff
@@ -1,19 +1,25 @@
 package service
 
 import (
+	"bytes"
 	"context"
-	"crypto/rand"
 	"fmt"
+	"hms/gateway/pkg/common"
```

Comment:
Keep it in order. 
1. standard+extended library packages
2. external
3. our internal

Reply:
\+